apps/experiments/models.py

```diff
@@ -626,6 +701,7 @@ def eligible_children(cls, team: Team, parent: Experiment | None = None):
                 .exclude(id__in=child_ids)
                 .exclude(id__in=parent_ids)
                 .exclude(id=parent.id)
+                .exclude(id__in=parent.versions.all())
```

Comment:
I think this would be more efficient as `working_version_id=parent.id`

Reply:
Ah yes, good catch thanks